src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -464,16 +561,16 @@ def generateBaseMesh(cls, region, options):
 
             # Create nodes
             nodeIdentifier = 1
-            lNodeIds = []
+            lNodeIdsL = []
```

Comment:
this variable needs to be called ``lNodeIds`` to be as Zohreh had it. Important to fix otherwise her code won't merge with yours.